Notify/WEB/ClientApp/src/components/UserMenu/NotificationListPage/Paging/Paging.js

```diff
@@ -0,0 +1,78 @@
+﻿import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import './Paging.css'
+
+import { setPageNumber } from '../../../../actions/notificationList'
+
+
+function Paging({ lastPage, selectPage, setPageNumber }) {
+
+    if (selectPage > lastPage && lastPage)
+        setPageNumber(lastPage)
+
+    function less() {
+        if (selectPage == 1)
```

Comment:
you can do this without this monstrous if clause
just pass different props depending on page number